main/course_info/about.php

```diff
@@ -125,6 +125,13 @@
 
 $subscriptionUser = CourseManager::is_user_subscribed_in_course($userId, $course->getCode());
 
+$courseInfo = api_get_course_info_by_id($courseId);
+$allowSubscribe = null;
+if ($courseInfo['subscribe'] == true || api_is_platform_admin()) {
```

Comment:
You can first set it false so you can avoid the else

```
$allowSubscribe = false;
if ($courseInfo['subscribe'] == true || api_is_platform_admin()) {
      $allowSubscribe = true;
}
```

Reply:
Fixed!